pages/linux/mkinitcpio.md

```diff
@@ -0,0 +1,36 @@
+# mkinitcpio
+
+> Generates one more initial ramdisk environments for booting the Linux kernel based on the specified preset(s).
```

Comment:
Perhaps this will read better?
```suggestion
> Generates initial ramdisk environments for booting the Linux kernel based on the specified preset(s).
```